src/background/extension.ts

```diff
@@ -49,6 +51,14 @@ export class Extension {
     static ALARM_NAME = 'auto-time-alarm';
     static LOCAL_STORAGE_KEY = 'Extension-state';
     constructor() {
+        /*
```

Comment:
What do you think about using `//` for inline comments and `/**/` for JSDoc comments? I think it's a common pattern.